xmtp_mls/migrations/2024-10-18-200341_create_key_value_store/up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS key_value_store (
```

Comment:
I'd love to understand what data we expect to put into this table over time? Is there no way we can have a better typed table for specific problems? Gets very hard to migrate a generic key/value store if the shape of the data changes.